lib/puppet/provider/ntp_config/nxapi.rb

```diff
@@ -75,22 +75,25 @@ def self.prefetch(resources)
   end # self.prefetch
 
   def exists?
-    return (@property_hash[:ensure] == :present)
+    @property_hash[:ensure] == :present
   end
 
   def validate
-    fail ArgumentError,
-      "This provider only supports a namevar of 'default'" unless @resource[:name].to_s == 'default'
+    unless @resource[:name].to_s == 'default'
+      fail ArgumentError, "This provider only supports a namevar of 'default'"
+    end
 
-    if @resource[:source_interface]
-      fail ArgumentError,
-        "The parameter 'source_interface' must not contain any spaces" \
-          if @resource[:source_interface] =~ /\s+/
+    return unless @resource[:source_interface]
 
-      fail ArgumentError,
-        "The parameter 'source_interface' must not contain any uppercase characters" \
-          if @resource[:source_interface] =~ /[A-Z]+/
+    if @resource[:source_interface] =~ /\s+/
+      fail ArgumentError, "The parameter 'source_interface' must not contain" \
+                          'any spaces'
     end
+
+    return unless @resource[:source_interface] =~ /[A-Z]+/
```

Comment:
It was hard to appease Rubocop in this method.  It doesn't like having multiple conditional blocks in a method and would rather you "return unless ...".

Are the end results of this change readable enough?